.config/semgrep.yaml

```diff
@@ -72,6 +72,13 @@ rules:
   pattern: tokio::time::sleep
   severity: WARNING
 
+- id: ban-futures-block-on
+  languages:
+    - rust
+  message: Use `tokio::runtime::Handle::current.block_on()` instead.
+  pattern: futures::executor::block_on
```

Comment:
finally
was this real cause of bug

Reply:
Nope.

Reply:
https://github.com/fedimint/fedimint/pull/4846 did